pkg/lib/bundlev2/generate.go

```diff
@@ -0,0 +1,54 @@
+package bundlev2
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+)
+
+// GenerateFunc generates a CSV for the bundle based on the resource within the bundle directory.
+// Input:
+// @bundleDir: Directory to the bundle is a required field
+// @outputDir: If not specified will write to bundle directory
+// @overwrite: If set to false, function throws error if a CSV exist.
```

Comment:
nit: I have not seen this commentary style in Go code before - generally comments are full sentences for auto-formatting purposes (godoc). See https://golang.org/doc/effective_go.html#commentary

Reply:
lol, i copied this style in the generate function in bundle v1. I'll change it back.